docs/multi-storage-transactions.md

```diff
@@ -17,7 +17,7 @@ level as long as the configuration is properly set for `multi-storage`. An examp
 configuration is shown as follows:
 
 ```properties
-# The storage is "multi-storage"
+# Multi-storage implementation is used for Consensus Commit
```

Comment:
```suggestion
# Consensus commit is required to use Multi-storage transactions
scalar.db.transaction_manager=consensus-commit

# Multi-storage implementation is used for Consensus Commit
```
I think it would be better to explain the Consensus Commit is required to use Multi-storage transactions to avoid misunderstanding.

For example, as I mentioned in another comment, we cannot use `JDBC transactions` with `Multi-storage transactions` even if the all underlying databases are JDBC databases.

What do you think?

Reply:
I agree. Of course, as the default value of `scalar.db.transaction_manager` is `consensus-commit`, we don't necessarily need to specify it explicitly. However, providing such explicit specification could serve as useful information for users.